### PR TITLE
Alternate Corporate Coats Wave 1

### PR DESCRIPTION
**Pull request by LovelyLophi**
# Description

Edited the corporate coat by Leonardo DaBepis, 

Thought we could have some more options for assistant gang colors. /j

---

<details><summary><h1>Media</h1></summary>
<p>

image

WAOW THEY LOOK SO COOOL RIGHT?

</p>
</details>

---

# Changelog

:cl:
- add: Added a number of new corporate coats to the Clothesmate.

### CODE REVIEW

**Comment by LovelyLophi** [comment]
I see the failure, give me a moment.

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
+  cost: 8050 #Einstein Engines
```
Comment (stellar-novas) [inline Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: Comments aren't needed on EE

Comment (LovelyLophi) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: > Comments aren't needed on EE afaik

until there is documentation I will simply play it safe for now.

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]:
```suggestion
  cost: 8050
```

Comment (LovelyLophi) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml]: so is this the standard @DEATHB4DEFEAT ?

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml]: We're EE

**Review by VMSolidus — APPROVED** [review]

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@
     sprite: Objects/Specific/Service/vending_machine_restock.rsi
     state: base
   product: CrateVendingMachineRestockClothesFilled
-  cost: 7250 #DeltaV
+  cost: 8050
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]:
```suggestion
  cost: 8250
```
8050 is weird looking :(

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml [hunk 3]
```diff
@@ -85,6 +85,11 @@
     ClothingOuterCoatLettermanRed: 2 # Nyano - Clothing addition
     ClothingOuterDenimJacket: 2 # DeltaV - Clothing addition
     ClothingOuterCorporateJacket: 2 # DeltaV - Clothing addition
+    ClothingOuterCsCorporateJacket: 2 # Einstein Engines - Clothing addition
+    ClothingOuterEECorporateJacket: 2 # Einstein Engines - Clothing addition
+    ClothingOuterHICorporateJacket: 2 # Einstein Engines - Clothing addition
+    ClothingOuterHMCorporateJacket: 2 # Einstein Engines - Clothing addition
+    ClothingOuterIdCorporateJacket: 2 # Einstein Engines - Clothing addition
```
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/clothesmate.yml:92]: Tags are fine here as it's in the middle of many other non-Wiz clothes.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 4]
```diff
@@ -541,3 +541,58 @@
...
+  - type: Sprite
+    sprite: Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi
+
+- type: entity
+  parent: ClothingOuterWinterCoat
+  id: ClothingOuterEECorporateJacket
+  name: Einstein Engines Corporate Jacket
+  description: A cozy jacket with the Einstein Engines logo printed on the back. Merchandise rewarded to stations with a safety factor of uhh... seven.
```
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:560]: Probably worth discussing this, I'm assuming by the other descriptions this is placeholder?

Wizard's Federation is only mentioned twice; the Wiz shuttle and the magic lamp (disabled on Delta forks)

File: Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json [hunk 5]
```diff
@@ -0,0 +1,18 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Leonardo DaBepis#4869 on discord / @leonardo-dabepis on Tumblr",
```
Comment (DangerRevolution) [inline Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json:4]: Did you draw this or did Phil?

**Review by DangerRevolution — CHANGES_REQUESTED** [review]
Very cool, they look nice; just a question on the meta.json's *and*

Going to ask for these to be localized in .ftl, which lets downstreams alter the descriptions of these without making merge conflicts for purposes such as translation for our Ukranian fork and altering the description for Syndicate-focused fork so it makes sense in their lore.

File: Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json (hunk 5 above)
Comment (LovelyLophi) [reply Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json:4]: Leo created the original jacket, I merely edited them to match the color schemes

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml (hunk 4 above)
Comment (LovelyLophi) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:560]: They all match the original coat's description.

File: Resources/Prototypes/Catalog/Cargo/cargo_vending.yml (hunk 2 above)
Comment (LovelyLophi) [reply Resources/Prototypes/Catalog/Cargo/cargo_vending.yml:36]: I like it better than 8000

File: Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json (hunk 5 above)
Comment (DangerRevolution) [reply Resources/Textures/Clothing/OuterClothing/WinterCoats/cs_corpo_jacket.rsi/meta.json:4]: you need to credit yourself for the edit

**Comment by LovelyLophi** [comment]
> Very cool, they look nice; just a question on the meta.json's _and_
> 
> Going to ask for these to be localized in .ftl, which lets downstreams alter the descriptions of these without making merge conflicts for purposes such as translation for our Ukranian fork and altering the description for Syndicate-focused fork so it makes sense in their lore.

@DangerRevolution I have no idea how to do FTL files so I think I'll leave that for other people if they want to pick this up. I'd prefer if there were some solid requirements for what a PR needs that I could read somewhere.

**Comment by DangerRevolution** [comment]
> > Very cool, they look nice; just a question on the meta.json's _and_
> > Going to ask for these to be localized in .ftl, which lets downstreams alter the descriptions of these without making merge conflicts for purposes such as translation for our Ukranian fork and altering the description for Syndicate-focused fork so it makes sense in their lore.
> 
> @DangerRevolution I have no idea how to do FTL files so I think I'll leave that for other people if they want to pick this up. I'd prefer if there were some solid requirements for what a PR needs that I could read somewhere.

Very real, localisation should probably be done by script when that's made one day. Standards *are* something we still have to decide on really, not really sure.

**Comment by DangerRevolution** [comment]
if you fix the credit by adding your name in i'll approve the review

**Review by DangerRevolution — APPROVED** [review]

**Review by FoxxoTrystan — COMMENTED** [review]
👍 

Just make sure all the tests passes.

**Comment by DangerRevolution** [comment]
> 👍
> 
> Just make sure all the tests passes.

they pass, just a microsoft issue

**Pull request merged**